ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleContent.vue

```diff
@@ -147,16 +150,16 @@ export default {
     onShowCodeClicked ({ identifier, levelNumber, hideCode = false }) {
       event.stopPropagation()
       event.preventDefault()
-      const relatedLevels = this.relatedLevels(levelNumber)
+      const relatedLevels = this.relatedLevels(levelNumber, identifier)
       for (const relatedLevel of relatedLevels) {
-        const identifier = relatedLevel.slug
+        const id = relatedLevel.slug
```

Comment:
Why is the id / slug confusion? Is `relatedLevel.slug` a slug or an id? Can we name it accordingly?

Reply:
ok